scripts/repl.js

```diff
@@ -284,7 +371,34 @@
   /*
    * Initialize the REPL
    */
-  var repl = new REPL();
+  var initialQuery = UriUtils.parseQuery();
+  if (initialQuery && initialQuery.short) {
```

Comment:
@sebmck Do you think it'd make sense to require all of these to be shortened urls?